<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FunWaveDialog</class>
 <widget class="QDialog" name="FunWaveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>函数波形</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resourcefile.qrc">
    <normaloff>:/Icon/Icon/wave-5.png</normaloff>:/Icon/Icon/wave-5.png</iconset>
  </property>
  <widget class="QPushButton" name="OutputOn_pushBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>82</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 1px solid #000000;
    border-radius: 5px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #dedede, stop: 0.5 #434343,
                                   stop: 0.51 #000000, stop: 1 #656a6d);
    color: #FFFFFF;
    font: bold 14px;
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #cfcccc, stop: 0.5 #333232,
                                   stop: 0.51 #000000, stop: 1 #585c5f);
    color: #00CC00;
}
QPushButton:flat {
    border: none;
}</string>
   </property>
   <property name="text">
    <string>OUTPUTON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StandBy_pushBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>82</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 1px solid #000000;
    border-radius: 5px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #dedede, stop: 0.5 #434343,
                                   stop: 0.51 #000000, stop: 1 #656a6d);
    color: #FFFFFF;
    font: bold 14px;
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #cfcccc, stop: 0.5 #333232,
                                   stop: 0.51 #000000, stop: 1 #585c5f);
    color: #00CC00;
}
QPushButton:flat {
    border: none;
}</string>
   </property>
   <property name="text">
    <string>STANDBY</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>241</width>
     <height>92</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_21">
      <item>
       <widget class="QLabel" name="FunWaveNo_Label">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="FunWaveNo_Cmb">
        <property name="minimumSize">
         <size>
          <width>85</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>85</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="FunWaveOhm_Label">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="text">
         <string>阻抗选择:</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="buddy">
         <cstring>DcWaveNo_Cmb</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="FunOhm_Cmb">
        <property name="minimumSize">
         <size>
          <width>85</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>85</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Andalus</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>241</width>
     <height>163</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_20">
        <item>
         <widget class="QLabel" name="FunWaveFreq_Label">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>输出频率:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="FunWaveFreq_Dsb">
          <property name="minimumSize">
           <size>
            <width>82</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>82</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="decimals">
           <number>5</number>
          </property>
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="FunWaveFreqUnit_Cmb">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QDial" name="FunWaveFreq_Dial"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_9">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>271</width>
     <height>261</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <item>
         <widget class="QLabel" name="FunWaveAmp_Label">
          <property name="minimumSize">
           <size>
            <width>71</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>输出幅度:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="FunWaveAmp_Dsb">
          <property name="minimumSize">
           <size>
            <width>82</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>82</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="decimals">
           <number>5</number>
          </property>
          <property name="maximum">
           <double>150.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="FunWaveAmpUnit_Cmb">
          <property name="minimumSize">
           <size>
            <width>78</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>78</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="smallDecimalPoint">
         <bool>true</bool>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Outline</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="FunWaveAmp_Dial"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resourcefile.qrc"/>
 </resources>
 <connections/>
</ui>
